tests/test_validation.py

```diff
@@ -790,3 +779,21 @@ def test_translate_unknown_oneoff_eerror():
             {'message': "Unknown type", 'validator': 'type'}
         ],
     }]) == {'example': [{'error': 'failed', 'index': 0}]}
+
+
+@pytest.mark.parametrize('email, expected_result', [('hurray@cool.gov', True), ('hurray@notcool.gov', False)])
```

Comment:
Do we need to cover subdomains here? like hurray@digital.cool.gov?

Reply:
Done!